Formula/vim.rb

```diff
@@ -19,6 +17,12 @@ class Vim < Formula
   LANGUAGES_OPTIONAL = %w[lua mzscheme python3 tcl]
   LANGUAGES_DEFAULT  = %w[perl python ruby]
 
+  if MacOS.version >= :mavericks
```

Comment:
The `ruby` and `perl` ones won't actually use custom versions unless we add the `:perl` and `:ruby` dependencies. Superenv prevents finding them, even from source.